packages/twenty-front/src/utils/workspace-url.helper.ts

```diff
@@ -0,0 +1,62 @@
+import { isDefined } from '~/utils/isDefined';
+
+export const twentyHostname = window.location.hostname;
+
+export const twentyHomePageHostname = `app.${twentyHostname}`;
+
+export const twentyHomePageUrl = `${window.location.protocol}//${twentyHomePageHostname}`;
+
+export const isTwentyHosting =
+  window.location.hostname.endsWith(twentyHostname);
+
+export const isTwentyHomePage =
```

Comment:
we usually do one file per util :p 

also naming are not fully clear to me and the busines logic does not seem right (app ==> defaultSubDomain). I know this will be refactored in upcoming PRs but adding a comment to revisit this

Reply:
we will need to add tests on thiese!
Also the naming of the file is not respecting FE conventions (looks like a BE util)

Reply:
It will change in the next PR:
https://github.com/twentyhq/twenty/blob/ef08ed785fb08abada82f76bf0ec2c6651dd734d/packages/twenty-front/src/modules/url-manager/hooks/useUrlManager.ts